Project 2: Building with Responsive Design, requesting for review

ixboy opened the pull request:
Hello, this is our second project on the program its a copy of the  https://thenextweb.com/  website. we used all the previous concepts about HTML like flexbox, float, and grid, we also made it to be responsive in 3 different screen size types. please review our work. any feedback will highly be appreciated.

Thank You.

FdI96 committed:
Update README.md

FdI96 committed:
Update README.md

FdI96 committed:
Update README.md

FdI96 committed:
Update README.md

FdI96 committed:
Add navbar

Adding the navbar divided in two parts.

FdI96 committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

FdI96 committed:
Add first part of the media querys breakpoints. (We have to finish it yet)

ixboy committed:
add the main landing div

ixboy committed:
create the news section

FdI96 committed:
Add Latest News section

ixboy committed:
add image folders with 4 logos

ixboy committed:
add git ignore

ixboy committed:
add main css folder

ixboy committed:
add .json pachage and started new section with four links for the middle part of the site

ixboy committed:
add the verticall section and ints divs

FdI96 committed:
Updating index.html

-Introducing the mobile version
-Add Mobile NavBar
-Hide FullScreen NavBar
-Hide Small things that aren't in the mobile version.
-Changing Latest news section.

FdI96 committed:
Update main.css

-Add style to the mobile version of the page:
      - Add style for the latest news section.
     - Add style for the Latest founding round section.
      -Fixed the positiong of the h tags.

FdI96 committed:
Add "Second News" section.

ixboy committed:
did the other news section, did the hardfork section with the nine inside elements

FdI96 committed:
Add "Latest Deals" section.

ixboy committed:
PUT THE IMAGE LOGOS

ixboy committed:
fixed conflict

FdI96 committed:
Add Ul in footer.

FdI96 committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

FdI96 committed:
Add the footer finished.

ixboy committed:
made the mid size screen media query, did news section

ixboy committed:
did the grid display and a bit of the bottom part

FdI96 committed:
Update index.html

-Add "Latest Founding" section fixed for medium screen size.
-Add "Image Group" section fixed for medium screen size
-Add first part of the "Latest deals" section for the medium screen size

ixboy committed:
add nav righten side log

ixboy committed:
ADD COLOR IN THE MAIN NAV

FdI96 committed:
Add "Latest Deals" section for the medium screen size.

FdI96 committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

FdI96 committed:
Updating index.html

-Finished Medium Screen Size
-Add features to the Full screen mode.

FdI96 committed:
Update index.html

ixboy committed:
Update README.md

ixboy committed:
Update README.md

ixboy committed:
deleted some comments

ixboy committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

ixboy committed:
Update README.md

add the page screenshot and the live demo link

ixboy committed:
add the page screenshot

ixboy committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

ioilmio commented on README.md:
```diff
@@ -1,2 +1,53 @@
+
+##  AUTHORS
+
+👤 **Author1**
```
include your name here instead of Author

ioilmio reviewed (changes requested):
# Hello @FdI96  @ixboy 👋🏼 
## Nice try on this one! Your project begins to take shape so keep it up like that! 👍🏼 

### You have to fix a couple of issues, let me list them:
#### Navigation Bar
- It should display different content depending on the screen size, it should contain links and not buttons;
![image](https://user-images.githubusercontent.com/51195150/100950779-6d53fd80-350d-11eb-8e08-101d789befcd.png)

- Medium version
 TNW logo appears at the leftmost part ([link to page](https://thenextweb.com/l))
- Small version 
 Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon)
![image](https://user-images.githubusercontent.com/51195150/100950819-8197fa80-350d-11eb-9311-f04114c4db21.png)


#### Top 3 articles section
- Large version 
 It uses an overlay gradient for the big image
![image](https://user-images.githubusercontent.com/51195150/100950856-9aa0ab80-350d-11eb-8433-b03966b808be.png)
- Medium version
 TNW logo disappears

#### Latest news section
- Large version
 It aligns 4 articles per row
- Medium version
 It aligns 3 articles per row

#### Latest Funding section
- Large version
 It aligns 4 articles per row

### Fix this margin issue please (large version)
![image](https://user-images.githubusercontent.com/51195150/100950693-2c5be900-350d-11eb-9352-eb7c3bdc804d.png)


#### Categories (Apps, Gear, Tech, etc) section
- Large version
It aligns 3 category boxes per row
- Medium version 
It aligns 2 category boxes per row

#### Latest deals section
- Large version
 It aligns 4 deals per row
![image](https://user-images.githubusercontent.com/51195150/100950986-dd628380-350d-11eb-9414-f69d19fdbb35.png)

- Medium version changes
 It aligns 3 deals per row

#### Footer
 It contains different social media icons  ([link to page](https://thenextweb.com/l))
![image](https://user-images.githubusercontent.com/51195150/100950894-b6a44d00-350d-11eb-826d-3abdafd282d5.png)
![image](https://user-images.githubusercontent.com/51195150/100950928-c0c64b80-350d-11eb-95b9-e443594ff295.png)



### General
- Refer to the Odin project for more detailed requirements about each section of the project
- Change font-family to adapt to the original page;
- Make all the linters checks pass
- Check better your top3 article section, I think your horizontal scroll comes from there...
- Check your medium screen size; I can't see the difference between large(1024px) and medium (768px) screen;
- Add your node_modules folder to .gitignore so it won't be pushed on GitHub(for what are you using it??)

### You can do it Team 💯  so don't be discouraged, is always your first review, and there's always room for improvements;
#### For any doubts or further info you can contact me on slack @illuminatosalvatore

FdI96 committed:
Update main.css for Full Screen Mode

-Add new order in "Latest News" section
-Add new order in "Second News" section (class name).
-Add  new order in  "Latest Deals" section

FdI96 committed:
Add gradients

ixboy committed:
did the latest funding section

ixboy committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

FdI96 committed:
Fix the logo flying at the medium screen size.

ixboy committed:
fixed the padding of the latest sec and changed the margin of the big cantainer

ixboy committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

FdI96 committed:
-Fixing NavBar for medium and full screen.

FdI96 committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

ixboy committed:
fixed the lattest funding sec correcly, put the pictures and aligned it propperly

FdI96 committed:
Updating main.css and index.html

-Add new social media logos at the footer

-Chaging padding from "Second News" section.

-Adding margin to the main tag and correcting the problems from the navbar

-Adding gap to the flex containers.

ixboy committed:
add pics on the folder and did the dorpdown

ixboy committed:
add the page screenshot

FdI96 committed:
Fixing dropwdowns from the navbar

FdI96 committed:
Fixing Bottom Part from the NavBar

Changed the background color to white to emphasize the words.

ixboy committed:
did few changes

ixboy committed:
add the hamburger menu and the logo

ixboy committed:
corrected some colors

ixboy committed:
add the page screenshot

ixboy committed:
add the orange color on the tittles

mimipeshy commented:
## STATUS: CHANGES REQUESTED 🚦 ⚠️
Hello @ixboy and @ioilmio 🤝 🤝 🤝
👍 This is good to make it this far, you have tried to implement some changes as required by the previous TSE.

- [x] However, before adding more changes, kindly work on **ALL** the changes requested by the previous  TSE. Please refer to the [Odin project ](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design)for more detailed requirements about each section of the project. Also, check out the original page (https://thenextweb.com/) for further details.

Regards
Peris Ndanu
Feel free to slack me if you have any questions.

FdI96 committed:
Fixing NavBar

FdI96 committed:
Adding png logo.

jubaan reviewed (changes requested):
# Status: Changes Required :recycle: 

There are e some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] The live preview link on the README links to a previous version of the website please update it, if I base my revision on that live preview, there are no changes made from the first review.

### [OPTIONAL]

These next observations are optional, but I must insist you work on them, the project right now looks sloppy and has these few issues that stop it from being a great project. I believe you can submit an awesome project :muscle: :sunglasses:.

- [ ] The media query dimension should be desktop version abote until 769px, tablet view 768px to 426px, mobile view 425 and below. 

- [ ] Element doesn't look as expected this might be related to the previous observation
![image](https://user-images.githubusercontent.com/25774518/101113680-85775b80-35a5-11eb-856f-985411e5ecd2.png)

- [ ] Element doesn't look as expected
![image](https://user-images.githubusercontent.com/25774518/101113716-99bb5880-35a5-11eb-8980-56f5f2f24849.png)

- [ ] Element has a margin that should not exist
![image](https://user-images.githubusercontent.com/25774518/101113794-bd7e9e80-35a5-11eb-8c89-aa765b204f13.png)

- [ ] Text and images should have a space between them
![image](https://user-images.githubusercontent.com/25774518/101113823-d4bd8c00-35a5-11eb-8ed3-788fabd1deef.png)

- [ ] Element have greater white space to the right, it should be equal to the left side.
![image](https://user-images.githubusercontent.com/25774518/101113905-fe76b300-35a5-11eb-82ae-4fadbb73fdc0.png)

### [Comments] :green_circle: 

- Use the media query inspector to observe the changes in the original website. 

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

ixboy committed:
fixed the width of the funding-sec, changed the footer color and fixed the screen sizes in media query section

ixboy committed:
fixed the space between text and images on the bottom section, fixed the unnecessary left padding on the ul

ixboy committed:
removed the old live demo link

ixboy committed:
changed the live demo link

ixboy committed:
add a new screenshot

layzrd reviewed (changes requested):
### Status: Require changes 🌳 ♲
Hi @ixboy and  @FdI96, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏   I appreciate your hard work in replicating thenextweb.com. You already fulfilled most of the requirements, kindly fix the following issues. 

[REQUIRED]

- [ ] Fix stylelint errors
- [ ] The navigation bar shouldn't have the TNW logo on large screens.

_Click the image to view it in full._

| Your Copy | Source |
| :---: | :---: |
|<img width="532" alt="Screen Shot 2020-12-04 at 6 33 21 PM" src="https://user-images.githubusercontent.com/38358081/101183019-f4b18600-365f-11eb-9c16-682662abbfb9.png">| <img width="1056" alt="Screen Shot 2020-12-04 at 6 42 51 PM" src="https://user-images.githubusercontent.com/38358081/101183398-86b98e80-3660-11eb-8df2-49d9dc7b99db.png">|

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

FdI96 committed:
Removing top logo in the Full Screen Mode

FdI96 committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

ixboy committed:
changed few linter error

FdI96 committed:
Updating main.css fixing linters.

FdI96 committed:
Fixing linters.

ixboy committed:
fixed more linters

FdI96 committed:
Fixing Errors.

FdI96 committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

FdI96 committed:
Fixing Image logo in the full screen

Srjnnnn reviewed (changes requested):
## Status: Changes Required :recycle: :stop_sign: 
Hello team! Great job on this project. :muscle: However before moving on, I'd like to mention some issues.
### Mandatory

- [ ] Please fix the linter issue. There shouldn't be a linter issue on your pull request when you submit for a review.

- [x] Kindly remove the arrow on the right and put a search icon instead.
![arrow](https://user-images.githubusercontent.com/33910796/101198062-f08f6380-3673-11eb-8cdd-d29c71f133e3.png)

- [x] Kindly remove the unnecessary padding around the footer.
![footer2](https://user-images.githubusercontent.com/33910796/101198135-07ce5100-3674-11eb-8eef-73ef8a95d08c.png)


### Optional

- [ ] You can write your names under the authors' section. 

- [ ] You can provide the [MIT license](https://github.com/git/git-scm.com/blob/master/MIT-LICENSE.txt) link on your project. That will make your repo more professional.

It is a great job till now and keep up the good job. :+1: 
Please fix these issues and submit for a re-review.
If you have any questions about the review, feel free to reach me via Slack. :100: 
_Eyüp Sercan UYGUR_

FdI96 committed:
Adding search bar full screen mode and configuring authors names.

FdI96 committed:
Adding stylelint fixer and removing some errors.

FdI96 committed:
Fixing global margin

FdI96 commented:
We already fix the requirements, but we are having a lot of issues to fix the StyleLint because when a change is done the same change appears as other errors that we have to fix, and so on.

mimipeshy commented:
## STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hey, team @ixboy and @FdI96  you guys did pretty well in this project 👏  👏  It almost looks like the original page. 

✔️ Good job on working on the TSE requirements

However, before we can mark it as complete lets fix three things. Don't feel discouraged when we keep requesting changes, we want you to be the best developers, now give me a smile 😄 

- [ ] The TNW logo is losing its pixels on the large version,  kindly check on that.  [link]( https://thenextweb.com/ )
View screenshot below 
![c](https://user-images.githubusercontent.com/33067256/101202709-b1184580-367a-11eb-8032-cf3a4d234349.PNG)
 

- [ ] Stylelint errors are still failing, ensure you run `npx stylelint` on localhost to check if there are any errors. If you get errors on localhost, your linters will fail too. Check this [link ](https://github.com/microverseinc/linters-config/tree/master/html-css)on how to setup linters

- [ ] Make the following titles a heading
![c](https://user-images.githubusercontent.com/33067256/101203507-d35e9300-367b-11eb-8e2e-2a999c21b59e.PNG)

- [ ] Add an MIT licence to make it more professional

Thats all 😌 , kindly submit another review when done.

Regards
Peris Ndanu
Feel free to slack me if you have any questions

ixboy committed:
correct lint

FdI96 committed:
Linters fixed.

ixboy committed:
did the logo

ixboy committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

ixboy committed:
reduced image size

FdI96 committed:
Fixing scrollbars

FdI96 committed:
Fixing NavBar scroll

ixboy committed:
fixed the top image

ixboy committed:
Merge branch 'Development' of https://github.com/FdI96/Building-with-responsive-desing into Development

ixboy committed:
fixed the left top logo

FdI96 committed:
Fixing mobile logo.

mahmoud717 reviewed (changes requested):
### Changes required ♻️ 

**Hello @ & @**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] please fix your linters errors and request another code review.

### Optional 🟢 

 

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

FdI96 committed:
FIxing Linter erros.

FdI96 committed:
Adding again "emoji-regex"

FdI96 committed:
Fixing Webhints

FdI96 committed:
Fixing Webhint and stylelint

mimipeshy commented:
## STATUS: APPROVED 💯 🥳 

Hey, team 👋 👋  congratulations on working so hard on this project, it's therefore approved. Go ahead and merge. All the best in your next projects.

Regards
Peris Ndanu
